### DISCUSSION TITLE
Encoding of prolation canons

### DISCUSSION THREAD

**Discussion by rettinghaus** [music-encoding/music-encoding#818] [Q&A]
How should a prolation canon with multiple voices notated as one layer be encoded? 
In the very famous example from the _Missa l'homme arme super voce musicales_ one could encode the notes in the three layers with `@sameas`. But what about the different mensurations? 
![Josquin_Agnus_Dei_mensuration_canon tiff](https://user-images.githubusercontent.com/7693447/113853422-eb673000-979d-11eb-8e37-8d462f9d952e.png)

There's a problem by using `staffDef`. You _could_  put all three signs as `mensur` in there, but then you have no connection to the correct layer. And interestingly `mensur` isn't allowed in `layerDef`. 

My proposal would be to allow `mensur` and `meter` as well as `keySig` and `clef` to be children of `layerDef`. 

This is somewhat connected to #697.

**Comment by craigsapp** [comment]
Here is the resolutio:

<img width="1147" alt="Screen Shot 2021-04-07 at 9 15 05 AM" src="https://user-images.githubusercontent.com/3487289/113899596-cfee2c00-9781-11eb-8eab-f34618518a23.png">

https://josquin.stanford.edu/work/?id=Jos0603

https://www.youtube.com/watch?v=OrZB-ioh5a4

Here is the Vatican manuscript of the canon (Capp.sist.197.f.0009v):

<img width="1047" alt="Screen Shot 2021-04-09 at 5 23 06 PM" src="https://user-images.githubusercontent.com/3487289/114252128-9de2f280-9958-11eb-96f1-a8b5b35c3203.png">

**Comment by craigsapp** [comment]
This example is also related to issue https://github.com/music-encoding/music-encoding/issues/686

The "3" mensuration from the manuscript version is the same as the "cut-c3" in the printed version (presumably by Petrucci?).  In this case it is weird to start a piece with a "3" since it usually occurs in the middle of a piece. It does mean sesquialtera (see the resolutio version), where the duration of the breve remains constant, but there are three semibreves instead of two.   In all cases I have in JRP it also seems to mean sesquialtera.

In the JRP scores, the "3" mensuration does not start at the beginning of a section (other than this one), and it can be preceded by an O or Cut-O mensuration.   In other words, only the "3" is displayed, but the "3" is implied to be in the context of the previous mensuration sign.   This mensuration is used interchangeably with colored notes (some parts may be colored while others switch to "3" mensuration), although I would have to examine what happens when the preceding mensuration is O or cut-O.

**Comment by kepper** [comment]
At ODD Friday, we agreed that the Mensural IG should have an eye on this. @martha-thomae, could you circulate this please? Thanks…

**Comment by martha-thomae** [comment]
Yes, of course. I will do that this coming week. Thanks!

**Comment by ahankinson** [comment]
I am going to move this to the discussions section.

**Comment by bwbohl** [comment]
2021-06-24 ODD Thursday: this discussion will be brought up at the IG mensural meeting.

**Reply by bwbohl** [reply]
It's already being discussed on the mensural Mailingliste
